libraries/Microsoft.Bot.Builder.Dialogs/Prompts/ChoicePrompt.cs

```diff
@@ -2,14 +2,95 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Prompts;
+using Microsoft.Bot.Builder.Prompts.Choices;
+using static Microsoft.Bot.Builder.Prompts.PromptValidatorEx;
 
 namespace Microsoft.Bot.Builder.Dialogs
 {
-    public class ChoicePrompt
+    public class ChoicePromptOptions : PromptOptions
```

Comment:
Code Comments.

Reply:
Missing code comments are going into the JavaScript first and then in here. As part of that process we make decide to make some of this internal.